vspace/src/main/webapp/WEB-INF/views/staff/spaces/customorder/customordering.html

```diff
@@ -0,0 +1,119 @@
+<html layout:decorate="~{layouts/main_staff}">
+<head>
+	<style>
+		.customOrderContainer {
+			background: white; 
+			padding: 25px;
+		}
+		.newOrder {
+			float: right;
+		}
+		.customOrderText {
+			padding-bottom:10px;
+		}
+		.successMsgAdded {
+			position: absolute;
+   			top: 8%;
+   			width: 65%;
+   			display:none;
+		}
+		.spaceDropdownSelect {
+			display: inline-block !important; 
+			height: auto; 
+			border: 1px solid; 
+			width: 11%; 
+			margin-right: 6px; 
+			position: absolute; 
+			right: 10%; 
+			z-index: 999; 
+			padding: 0;
+			border-radius: 4px;
+		}
+		.customOrderList:hover, .spaceDropdownOption:hover {
+			background: #D3D3D3;	
+		}
+		.spaceListItem {
+			vertical-align: text-top;
+		}
+		.spaceListItem:hover {
+			font-weight: bold;
+		}
+	</style>
+	<script>
+		$(document).ready(function() {
+			if(localStorage.getItem("prevUrl")) {
+				if(localStorage.getItem("prevUrl").includes("/space/order/add")) {
+					$("#successText").text("Custom Order has been successfully added.");
+				} else if(localStorage.getItem("prevUrl").includes("/space/order/deletedOrder")) {
+					$("#successText").text("Custom Order has been successfully deleted.");
+				}
```

Comment:
what is that if statement for? Is that for after the redirect (after adding or deleting)? In that case flash attributes should be used (`RedirectAttributes` injected in the controller method)